tensorbay/client/job.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python3
+#
+# Copyright 2021 Graviti. Licensed under MIT License.
+#
+
+"""Basic structures of asynchronous jobs."""
+
+from typing import Any, Dict, Optional
+
+
+class GetLogResponse:
```

Comment:
Why we need a new class for log?
Shall we use python `StringIO`? If not, should this new class follows the `IO` protocol?